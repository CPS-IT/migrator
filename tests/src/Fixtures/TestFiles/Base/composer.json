{
	"name": "cpsit/migrator-test",
	"require": {
		"php": "~8.1.0 || ~8.2.0 || ~8.3.0"
	},
	"require-dev": {
		"phpunit/phpunit": "^9.5"
	}
}
