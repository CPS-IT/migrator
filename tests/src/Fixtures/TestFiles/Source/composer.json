{
	"name": "cpsit/migrator-test",
	"require-dev": {
		"phpunit/phpunit": "^9.5"
	}
}
