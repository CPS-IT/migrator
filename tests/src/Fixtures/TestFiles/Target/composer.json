{
	"name": "cpsit/migrator-test",
	"require-dev": {
		"phpunit/phpunit": "^10.0"
	}
}
