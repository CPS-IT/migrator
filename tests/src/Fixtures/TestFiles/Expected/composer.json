{
	"name": "cpsit/migrator-test",
	"require": {
		"php": "~8.1.0 || ~8.2.0"
	},
	"require-dev": {
		"phpunit/phpunit": "^10.0"
	}
}
